src/Symfony/Component/Form/ButtonBuilder.php

```diff
@@ -503,6 +509,28 @@ public function setAutoInitialize($initialize)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function setPosition($position)
+    {
+        if ($this->locked) {
+            throw new BadMethodCallException('The config builder cannot be modified anymore.');
+        }
+
+        if (is_string($position) && ($position !== 'first') && ($position !== 'last')) {
+            throw new InvalidConfigurationException('If you use position as string, you can only use "first" & "last".');
```

Comment:
`When using position as a string, the only supported values are "first" and "last".`